src/main/scala/org/youshuren/endpoint/Routes.scala

```diff
@@ -0,0 +1,60 @@
+package org.youshuren.endpoint
+
+import akka.event.Logging
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import org.youshuren.database.Database
+import org.youshuren.model.Book
+import org.youshuren.codec.JsonFormats._
+import Implicits._
+import scala.util.{Failure, Success}
+
+object Routes {
+
+  val Books: String = "books"
+  val OnlyAvailableQueryParam: Symbol = 'only_available
+  val BookNameQueryParam: Symbol = 'book_name
+  val log = Logging(system, this.getClass)
+
+  lazy val routes: Route = {
+
+    path(Books / Remaining) { bookName =>
```

Comment:
we can make the route more readable

Reply:
could you provide some example here?

Reply:
please see another PR https://github.com/zailushangde/Youshuren/pull/3